config/truflationFeed.json

```
@@ -0,0 +1,12 @@
+{
+  "answerValueCeiling": 1815634312,
+  "answerValueFloor": 1615634312,
```

```suggestion
  "desiredAnswerValue": 1815634312,
  "answerValuePercentageDriftTolerance": 10,
```
Instead of having the upper and lower limits, request mid value and % of tolerance.